src/client/src/shell/openFin/openFin.ts

```diff
@@ -56,6 +59,12 @@ export class OpenFinLimitChecker {
     fin.desktop.main(() => {
       fin.desktop.InterApplicationBus.addSubscribeListener(this.setLimitCheckSubscriber)
       fin.desktop.InterApplicationBus.addUnsubscribeListener(this.removeLimitCheckSubscriber)
+      fin.desktop.InterApplicationBus.subscribe(LIMIT_CHECKER_UUID, null, LIMIT_CHECKER_STATUS_TOPIC, (m, u, n) => {
```

Comment:
please can you rename the parameters `(m, u, n)` with meaningful names?

Also if you don't need `u` or `n` it is conventional to replace with `_` to indicate you ignore them ([see this](https://stackoverflow.com/questions/41085189/using-underscore-variable-with-arrow-functions-in-es6-typescript)) or in this case, since they are last in the list of parameters, just omit them directly on the handler:
```typescript
fin.desktop.InterApplicationBus.subscribe(LIMIT_CHECKER_UUID, null, LIMIT_CHECKER_STATUS_TOPIC, message => { ... }
```